.github/workflows/pull.yml

```diff
@@ -110,3 +110,21 @@ jobs:
     uses: ./.github/workflows/_unittest.yml
     with:
       docker-image: executorch-ubuntu-22.04-clang12
+
+  test-arm-backend-delegation:
+    name: test-arm-backend-delegation
+    uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
+    with:
+      runner: linux.2xlarge
+      docker-image: executorch-ubuntu-22.04-clang12
```

Comment:
You would need to use the new ARM image here once it's ready

Reply:
Yes!  Please let me know when it's ready.  So far I use the main image just for debugging

Reply:
@huydhn ETA when it will be ready? @digantdesai if Arm really want to enable it asap, we can enable it by installing the their simulator and toolchains on the fly like this.